packages/frontend/src/components/Explorer/CustomDimensionModal/CustomSqlDimensionModal.tsx

```diff
@@ -67,11 +71,22 @@ export const CustomSqlDimensionModal: FC<{
                     return null;
                 }
 
-                return customDimensions?.some(
-                    (customDimension) =>
-                        customDimension.id === customDimensionId,
-                )
-                    ? 'Dimension with this label already exists'
+                const tableCalculationsIds = tableCalculations.map(
+                    (tc) => tc.name,
+                );
+
+                const customDimensionsIds =
+                    customDimensions?.map(
+                        (customDimension) => customDimension.id,
+                    ) ?? [];
+
+                const isInvalid = [
+                    ...tableCalculationsIds,
+                    ...customDimensionsIds,
+                ].some((id) => id === customDimensionId);
+
+                return isInvalid
+                    ? 'Dimension/Table calculation with this label already exists'
```

Comment:
The problem also exists when I create a custom dimension and then a table calculation with the same name.

Reply:
Done here: https://github.com/lightdash/lightdash/pull/10140/commits/244720998940d9af3fd5d185a1a3b67f583e8810

also trimmed and lowercased on both ends of the comparison: https://github.com/lightdash/lightdash/pull/10140/commits/980677ce58ca092b4ce67e56230b15c341af881c